cli/manifest_parser/libs_by_loc_map.go

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2014-2019 Cesanta Software Limited
+// All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package manifest_parser
+
+import (
+	"sync"
```

Comment:
please add a blank line to separate standard library imports from local

Reply:
Done!  Also applied gofmt and shortened error handling a bit as discussed over IM.